src/percetto.cc

```diff
@@ -23,9 +23,9 @@
     defined(_M_AMD64)
   #define HAS_RDTSC
 #endif
-#if defined(_M_ARM) || defined(_M_ARMT) || defined(__arm__) || \
-    defined(__thumb__) || defined(__aarch64__)
-  // TODO(jbates): support ARM CPU counter
+#if defined(__aarch64__)
+#error asdfasdfasdf
```

Comment:
asdf?

Reply:
Thanks for reviewing :-) I removed the testing code!